appserver/admingui/jms-plugin/src/main/java/org/glassfish/admingui/plugin/jms/JmsHandlers.java

```diff
@@ -153,17 +151,14 @@ public static void getPhysicalDestination(HandlerContext handlerCtx) {
             GuiUtil.handleException(handlerCtx, ex);
         }
 
-
         handlerCtx.setOutputValue("destData", valueMap);
     }
 
-    @Handler(id = "getPhysicalDestinationStats",
-    input = {
-        @HandlerInput(name = "name", type = String.class, required = true),
-        @HandlerInput(name = "type", type = String.class, required = true),
-        @HandlerInput(name = "target", type = String.class, required = true)},
```

Comment:
I think that it is more readable to have the output attribute on a  new line, but I can survive even this.